.github/workflows/changelog.yml

```
@@ -1,35 +1,31 @@
-name: Update CHANGELOG
+name: Changelog CI
 
 on:
-  release:
-    types: [created]
+  pull_request:
+    types: [ opened ]
+
   workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Set Release Version'
+        required: true
+  push:
+    branches:
+      - dev
```

just a missed space here
```suggestion
  pull_request:
    types: [ opened ]

  workflow_dispatch:
    inputs:
      release_version:
        description: 'Set Release Version'
        required: true
   
  push:
    branches:
      - dev
```